controllers/gcpmachine_controller.go

```diff
@@ -365,6 +365,12 @@ func (r *GCPMachineReconciler) reconcileLBAttachment(machineScope *scope.Machine
 		return errors.Wrapf(err, "failed to add instance to group")
 	}
 
+	computeSvc := compute.NewService(clusterScope)
```

Comment:
Can we add a TODO to remove this and debug why we'd need to reconcile it here?

Reply:
will do!

Reply:
Or an issue, let's merge this for now and debug later :)

Reply:
cool opening